build.gradle

```diff
@@ -16,6 +16,7 @@ repositories {
 
 dependencies {
     testCompile group: 'junit', name: 'junit', version: '4.12'
+    implementation 'org.json:json:20190722'
```

Comment:
`implementation group: 'org.json', name: 'json', version: '20190722'` to preserve the style?

Reply:
good idea, will do :+1: